MinervaBot.py

```diff
@@ -33,10 +36,16 @@
 password.send_keys(inputpassword, Keys.RETURN)
 
 # Acha o botao de fechar o popup e o fecha
-driver.implicitly_wait(10)  # Aguarda o modal dialog estar disponivel
-popup = driver.find_element_by_css_selector(
-    '.modal-footer > button[data-dismiss="modal"]')
-popup.send_keys(Keys.RETURN)
+try:
+    WebDriverWait(driver, 10).until(
+        EC.presence_of_element_located((By.ID, 'myModal'))
+    ) # Aguarda o popup estar disponivel
+except:
```

Comment:
It's better to do a `except TimeoutException` than to just use a bare `except`.

I guess we can also do some error handling, at least logging it and/or printing out should be fine. How do you feel about that?

Reply:
Good catch! From my understanding, a `TimeoutException` will only be raised when/if they remove the popup or change its ID, that's why I'm `pass`ing the exception.

---

About error handling, I'm planning another PR to deal with 1) unsuccessful logins and 2) no borrowed books.

In the first case maybe we should fallback to inputting the login and password via CLI. I'm also trying to think of a better way to store login info.

In the second case, I believe doing nothing or just printing a message would be the better choice.

Reply:
LGTM, I'm also interested in improving the login information processing. Thanks!